staging/src/k8s.io/apiserver/pkg/util/logs/logs.go

```diff
@@ -77,3 +87,75 @@ func GlogSetter(val string) (string, error) {
 	}
 	return fmt.Sprintf("successfully set glog.logging.verbosity to %s", val), nil
 }
+
+var (
+	outputFile *os.File
+	logTeeWG   sync.WaitGroup
+)
+
+// InitLogFile sets up the log file, and redirects stdout & stderr to the file. This must be called
+// before any logging, otherwise those log lines will not be captured.
+func initLogFile() {
+	if !flag.Parsed() {
+		os.Stderr.WriteString("ERROR: InitLogFile before flag.Parse")
+		return
+	}
+
+	if *logFile == "" {
+		// If log-file is unset, nothing to do.
+		return
+	}
+
+	// If log-file is set, glog's logtostderr MUST be set to capture logs.
+	if f := flag.Lookup("logtostderr"); f != nil {
+		if f.Value.String() != "true" {
```

Comment:
why stringify?

Reply:
I'm not sure what you mean? I think this is the only way to interact with flag values.